frontend/package.json

```diff
@@ -36,6 +36,7 @@
     "is-docker": "^3.0.0",
     "jsonschema": "^1.4.1",
     "lodash-es": "^4.17.21",
+    "markdown-to-jsx": "^7.3.2",
```

Comment:
This can be removed as `frontend/src/common/MarkdownRenderer.tsx` has been changed in my branch